--- kvm-install.sh	2013-08-08 16:47:11.989338125 -0400
+++ kvm-install.sh.dc0pkvm-hv02	2013-11-01 19:24:58.543174118 -0400
@@ -14,8 +14,8 @@
 }
 
 # constants
-WEBROOT="http://192.168.32.39/centos"
-VMDIR=/san/virtual-machines
+WEBROOT="http://192.168.1.243/centos"
+VMDIR=/var/lib/libvirt/images
 
 # settings
 OS="$3"
@@ -30,9 +30,9 @@
 if [ `whoami` != 'root' ]; then
   echo "You must be root"; exit 1
 fi
-if [[ `hostname` != 'kvm-01' && `hostname` != 'kvm-02' && `hostname` != 'kvm-03' ]]; then
-  echo "Run on kvm"; exit 1
-fi
+#if [[ `hostname` != 'kvm-01' && `hostname` != 'kvm-02' && `hostname` != 'kvm-03' ]]; then
+#  echo "Run on kvm"; exit 1
+#fi
 
 # verification
 if [[ "$VMNAME" == "" || "$VMIP" == "" ]]; then usage; fi
@@ -55,17 +55,17 @@
     virt-install --name ${VMNAME} --ram=${VMRAM} --vcpus=${VMCPUS} \
     --os-type=linux --os-variant=rhel5.4 --accelerate --hvm \
     --disk path=${VMDIR}/${VMNAME}.img,size=${VMSIZE},format=qcow2,cache=writeback \
-    --network=bridge:br3 --location=${WEBROOT}/5/os/i386/ \
-    --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-5.cfg ksdevice=br3 \
-    ip=${VMIP} netmask=255.255.254.0 dns=192.168.32.46 gateway=192.168.32.10 \
+    --network=bridge:br0.2 --location=${WEBROOT}/5/os/i386/ \
+    --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-5.cfg ksdevice=br0.2 \
+    ip=${VMIP} netmask=255.255.255.0 dns=192.168.1.3 gateway=192.168.1.3 \
     console=ttyS0"
   else
     virt-install --name ${VMNAME} --ram=${VMRAM} --vcpus=${VMCPUS} \
     --arch=x86_64 --os-type=linux --os-variant=rhel5.4 --accelerate --hvm \
     --disk path=${VMDIR}/${VMNAME}.img,size=${VMSIZE},format=qcow2,cache=writeback \
-    --network=bridge:br3 --location=${WEBROOT}/5/os/x86_64/ \
-    --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-5x86_64.cfg ksdevice=br3 \
-    ip=${VMIP} netmask=255.255.254.0 dns=192.168.32.46 gateway=192.168.32.10 \
+    --network=bridge:br0.2 --location=${WEBROOT}/5/os/x86_64/ \
+    --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-5x86_64.cfg ksdevice=br0.2 \
+    ip=${VMIP} netmask=255.255.255.0 dns=192.168.1.3 gateway=192.168.1.3 \
     console=ttyS0"
   fi
   ;;
@@ -75,9 +75,9 @@
   virt-install --name ${VMNAME} --ram=${VMRAM} --vcpus=${VMCPUS} \
   --arch=x86_64 --os-type=linux --os-variant=rhel6 --accelerate --hvm \
   --disk path=${VMDIR}/${VMNAME}.img,size=${VMSIZE},format=qcow2,cache=writeback \
-  --network=bridge:br3 --location=${WEBROOT}/6/os/x86_64/ \
-  --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-6.cfg ksdevice=br3 \
-  ip=${VMIP} netmask=255.255.254.0 dns=192.168.32.46 gateway=192.168.32.10 \
+  --network=bridge:br0.2 --location=${WEBROOT}/6/os/x86_64/ \
+  --nographics --extra-args="ks=${WEBROOT}/ks/generic-vm-6-64.cfg ksdevice=br0.2 \
+  ip=${VMIP} netmask=255.255.255.0 dns=192.168.1.3 gateway=192.168.1.3 \
   console=ttyS0"
   ;;
  *)
